README.md

```diff
@@ -98,10 +98,11 @@ calling etc.*
 Antonis Tsipaliokas • [GitHub](https://github.com/kokeroulis)  
 Daniel Sobczak • [GitHub](https://github.com/DanielSobczak) (remote)  
 
-##### Partnerships squad
+##### Digital first primary care squad
```

Comment:
You missed the `(South Kensington)` bit

Reply:
Have I? We're under `Clinical care experience tribe (South Kensington)`, none of these squads mention the location. It's in the tribe's label.

Reply:
You're right, ignore me 😄